tsconfig.json

```diff
@@ -1,11 +1,13 @@
 {
-	"extends": "@sindresorhus/tsconfig/tsconfig.json",
-	"include": [
-		"source",
-		"test"
-	],
-	"compilerOptions": {
-		"module": "CommonJS",
-		"outDir": "dist"
-	}
+  "extends": "@sindresorhus/tsconfig/tsconfig.json",
+  "include": ["source"],
+  "exclude": [
+    "**/*.test.*",
+    "**/test.*"
+  ],
+  "compilerOptions": {
+    "rootDir": "source",
```

Comment:
I think with my removal of the test folder this option is superfluous now? Maybe remove it to keep it simpler again?

Reply:
Sure.